clients/da-vinci-client/src/main/java/com/linkedin/davinci/kafka/consumer/ConsumerPoolType.java

```diff
@@ -0,0 +1,16 @@
+package com.linkedin.davinci.kafka.consumer;
+
+public enum ConsumerPoolType {
+  /**
+   * The following pool types are being used when the dedicated AA/WC leader consumer pool feature is enabled.
+   */
+  AA_WC_LEADER_POOL, // For AA/WC leader
+  REGULAR_POOL, // For other kinds of workload except AA/WC leader
```

Comment:
Nits about comment: seems REGULAR_POOL will be used for both DEFAULT and  AA_OR_WC_LEADER_DEDICATED strategy.

Reply:
Sure